authui/src/authflowv2/components/close-btn.css

```diff
@@ -0,0 +1,15 @@
+.close-btn {
+  @apply cursor-pointer;
+  
+  /* Make button clickable area at least 44px x 44px */
+  @apply p-5.5 -m-5.5;
+  transform: translateX(-0.2rem); /* FIXME: Figure out why button shifted right after applying above */
```

Comment:
What?

Reply:
ref [this SO answer](https://stackoverflow.com/a/20327676/19287186)

Any better suggestion on `enlarging button clickable area`? Maybe `absolute` / `relative` position with `positive padding + negative margin`?